LEGO1/omni/include/mxmemorypool.h

```diff
@@ -0,0 +1,86 @@
+#ifndef MXMEMORYPOOL_H
```

Comment:
Can we add annotations where possible, and decomp size asserts? Thank you!

Reply:
The closest logical place for annotations would be mxstreamer.cpp. I had to typedef the memory pools because I guess the assert didn't like the double template. This look ok naming-wise?
```cpp
DECOMP_SIZE_ASSERT(MxMemoryPool64, 12);
DECOMP_SIZE_ASSERT(MxMemoryPool128, 12);
DECOMP_SIZE_ASSERT(MxBitset<22>, 4);
DECOMP_SIZE_ASSERT(MxBitset<2>, 4);
```

What annotations should I add? `// SIZE`? The two classes are always 12 and 4 bytes where used, but they both have variable size. The only two functions there to annotate are the two destructors on the memory pools.

Reply:
Member offsets should be mostly possible. Also the Reference. Anything that is template dependent we can just ignore

Reply:
naming looks good, for the size values we should use hex instead of decimal

Reply:
Ah yes, I missed those